src/style.css

```
@@ -0,0 +1,25 @@
+* {
+  margin: 0;
+}
```

```suggestion
body {
  margin: 0;
}
```
its forbidden to use anything except box-sizing in `*`